doc/classes/TreeItem.xml

```diff
@@ -246,6 +254,12 @@
 				Returns the previous sibling TreeItem in the tree or a null object if there is none.
 			</description>
 		</method>
+		<method name="get_prev_int_tree">
+			<return type="TreeItem" />
+			<param index="0" name="wrap" type="bool" default="false" />
+			<description>
+			</description>
```

Comment:
Needs description. You can copy-paste it from `get_next_in_tree`.
Also I now noticed typo in method's name.

Reply:
Oh, looks like I accidentally inserted that t in the binding name later since I remember writing this description. Fixed.